pyproject.toml

```diff
@@ -101,17 +102,12 @@ include-package-data = false
 
 [tool.uv]
 environments = ["sys_platform == 'linux'", "sys_platform == 'windows'"]
-
 prerelease = "if-necessary-or-explicit"
-
+index-strategy = "unsafe-best-match"                                    # Needed for TRT-LLM
 
```

Comment:
Why is this added? What is the significance?

Reply:
This is for uv to install Tensorrt llm